cmd/cosign/cli/trustedroot/trustedroot.go

```diff
@@ -219,3 +233,11 @@ func getPubKey(path string) (*crypto.PublicKey, string, []byte, error) {
 
 	return &pubKey, keyID, idBytes, nil
 }
+
+func getCheckpointID(origin string, key crypto.PublicKey) (string, []byte, error) {
+	_, id, err := note.KeyHash(origin, key)
+	if err != nil {
+		return "", nil, err
+	}
+	return hex.EncodeToString(id), id, nil
```

Comment:
This should be base64 encoded - https://github.com/sigstore/root-signing-staging/blob/b2f403d77cfc959af5a3359fdcb7b57a06fd5f6d/targets/trusted_root.json#L29

Hex encoding was for Rekor v1

Reply:
Ah yes, I missed this was returning the non-encoded bytes as well. Nvm!